packages/generator-langium/templates/cli/bin/cli.js

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env node
 
-import '../out/cli/main';
+import main from '../out/cli/main.js';
+main();
```

Comment:
Minor: Please at a new line to the end of this file.

Reply:
@msujew updated the PR